conf/application.conf

```diff
@@ -59,3 +59,13 @@ logger.play=INFO
 
 # Logger provided to your application:
 logger.application=DEBUG
+
+# Hub configuration
+# ~~~~~
+
+# Email address of site administrator
+hub.admin.email="rrodgers@mit.edu"
+
+# URL of email service, and API key to use it
+hub.email.url="https://api.mailgun.net/v2/topichub.mailgun.org/messages"
+hub.email.apikey=${MAILGUN_API_KEY}
```

Comment:
It may be worth building in some sort of application init check to ensure this (and other future required ENV values) is/are either set or populated with a dummy variable as it may not be clear to future users what's going on when the app crashes on startup. I'd probably just open a ticket though as for now it's just us and we know what's going on.